server/routes/actions/actions_routes.py

```
@@ -0,0 +1,395 @@
+    "name": "name = %s",
+    "description": "description = %s",
+    "instructions": "instructions = %s",
+    "trigger_type": "trigger_type = %s",
+    "trigger_config": "trigger_config = %s",
+    "mode": "mode = %s",
+    "enabled": "enabled = %s",
+    "updated_at": "updated_at = %s",
+}
+
+
+def _validate_trigger_config(body):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_validate_trigger_config`

(ANN202)